src/layouts/MainContact.astro

```diff
@@ -1,45 +1,16 @@
 --- 
-import BasicMeta from '../components/BasicMeta.astro'
-import Header from '../components/Header.astro'
-import Footer from '../components/Footer.astro'
-import Global from '../components/Global.astro'
-const {title} = Astro.props;
+import MainLayout from './Main.astro'
+const { title } = Astro.props;
 ---
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <BasicMeta title={title} />
-    <Global />
-</head>
-
-<body>
-    <Header />
-    <main>
-        <slot />
-    </main>
-    <Footer />
-    <script data-goatcounter="https://vivic.goatcounter.com/count" async src="//gc.zgo.at/count.js"></script>
-</body>
-</html>
-
-<style>
-body {
-    color: var(--white);
-    min-height: 100vh;
-    background-image: radial-gradient(88% 140% at 120% -5%, #482979, #40256b, #38215e, #301d51, #291943, #211536, var(--dark) 80%);
-    background-size: cover;
-    background-repeat: no-repeat;
-    background-color: var(--dark);
-    margin: 0;
-}
-
-main {
-    font-family: var(--mainfont);
-    min-height: calc(100vh - 340px);
-    margin: 8px;
-    padding-top: 150px
-}
+<MainLayout title={title}>
+	<slot/>
+</MainLayout>
 
+<style global>
+	main {
+		padding-top: 150px;
+		min-height: calc(100vh - 340px);
+	}
```

Comment:
The newfound smallness of this layout really conveys its purpose: the Main layout, but centered.

Reply:
Yah doing this makes sense. Forsure again I did a truly dumb move over and now improvements can be made lol